src/main/resources/migrations/00033_compound_index_transaction_id_created_date.sql

```diff
@@ -0,0 +1,5 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:compound_index_transaction_id_created_date runInTransaction:false
+
+CREATE INDEX CONCURRENTLY IF NOT EXISTS id_created_date_idx ON transaction USING btree(id, created_date);
```

Comment:
Can the index be named like `transaction_....._idx` as we do for rest of the indexes on transaction table.